### ISSUE TITLE
Number of epochs for segmentation and cubes classification

### ISSUE THREAD

**Issue by DandiC** [ranahanocka/MeshCNN#24]
Hello @ranahanocka, I've noticed that the number of epochs when running the training scripts for segmentation and cubes classification is 2100, while the number of epochs for SHREC classification is 200. I am curious about this big difference, especially since the other parameters of the networks are so similar. 

In my experiments, it seems like the testing accuracy is stable way before reaching 2100 epochs, so I was wondering if this is a small bug (--niter_decay is only specified in SHREC classification but not for the other experiments). If it isn't, I would appreciate if you could share your insights about why SHREC does not need that many training iterations compared to the other datasets.

Thanks!

**Comment by ranahanocka** [comment]
Hi @DandiC ,

Yes, you are correct we don't actually run the segmentation for 2100 epochs (it is just the default in the code, and I didn't change it). I think we got to a very good test accuracy on human seg after only around 30 epochs or so.
